datafusion/sqllogictest/test_files/copy.slt

```diff
@@ -139,20 +160,11 @@ select * from validate_single_json;
 1 Foo
 2 Bar
 
+# Error cases:
+
 # Copy from table with options
-query IT
+query error DataFusion error: Invalid Option: Found unsupported option row_group_size with value 55 for JSON format!
```

Comment:
As discussed in #6569, this query now throws an error since row_group_size does not make sense in the context of writing JSON files.